templates/zerver/about.html

```diff
@@ -1,124 +1,181 @@
-
-<p>
-    Zulip was originally developed by Zulip, Inc., a small startup in
-    Cambridge, Massachusetts.  Zulip, Inc. was founded by the MIT team
-    that previously created
-    <a href="https://www.ksplice.com">Ksplice</a>, software for
-    live-patching a running Linux kernel.  Zulip was inspired by
-    the <a href="https://barnowl.mit.edu/">Barnowl</a> client for
-    the <a href="https://en.wikipedia.org/wiki/Zephyr_(protocol)">Zephyr</a>
-    protocol, and the incredible community that Zephyr supported at MIT.
-</p>
-
-<p>
-    Zulip, Inc. was acquired by Dropbox in early 2014, while the product
-    was still in private beta.  Zulip's beta
-    users <a href="https://www.recurse.com/blog/90-zulip-supporting-oss-at-the-recurse-center">loved
-    Zulip's unique user experience</a> and continued using it, despite
-    the fact that the product was not being actively developed.  After a
-    year and a half, Dropbox generously decided to release Zulip as open
-    source software so that Zulip's users could continue enjoying the
-    software.
-</p>
-
-<p>
-    As a result, the first time the public had the opportunity to use
-    Zulip was when Dropbox
-    <a href="https://blogs.dropbox.com/tech/2015/09/open-sourcing-zulip-a-dropbox-hack-week-project/">released
-    Zulip as open source software</a> in late 2015.  The open sourcing
-    announcement was very popular, staying at the top of
-    both <a href="https://news.ycombinator.com/item?id=10279961">Hacker
-    News</a>
-    and <a href="https://www.reddit.com/r/programming/comments/3me9qp/dropbox_has_open_sourced_zulip_group_chat_software/">the
-    programming subreddit</a> for an entire day.
-</p>
-
-<p>
-    Zulip was open sourced with the complete version control history
-    intact because 10 Zulip users visited Dropbox for a full week to
-    help with the technical work.  The Zulip community is incredibly
-    grateful to both Dropbox and those enthusiastic early users for
-    making the Zulip open source project possible.
-</p>
-
-<p>
-    By the end of 2015, the open source project was already going strong
-    with a community of dozens of developers around the world.  By
-    late 2016, <a href="https://en.wikipedia.org/wiki/Zephyr_(protocol)">more
-    than 150 people from all over the world</a> have contributed
-    <a href="https://github.com/zulip/zulip/pulls">almost 1000
-    improvements</a> to the software, and the Zulip project is moving
-    faster than when the original startup employed 11 full-time
-    engineers.  As of mid-2017, those numbers have ballooned to almost
-    3000 pull requests written by nearly 300 developers.
-</p>
+{% include 'zerver/landing_nav.html' %}
+
+<div class="portico-landing why-page no-slide">
+    <div class="hero">
+        <h1 class="center">{% trans %}A little about us{% endtrans %}</h1>
+        <p>Lorem ipsum filler text. Alabama alaska arizona arkansas. California, colorado, connecticut. Delaware florida.</p>
```

Comment:
The lorem ipsum? Yes, I'm looking for @timabbott or @rishig or someone to throw in some real text, which I'll then fixup into my branch.

I'll add a WIP tag so not to confuse anyone.